docs/static/releasenotes.asciidoc

```diff
@@ -1,4 +1,118 @@
 [[releasenotes]]
 == Release Notes
-coming[9.0.0]
 
+* <<logstash-9-0-0-beta1,Logstash 9.0.0-beta1>>
+
+[[logstash-9-0-0-beta1]]
+=== {ls} 9.0.0-beta1 Release Notes
+
+==== Breaking changes
```

Comment:
For Breaking Changes, until we have a final decision on how we are going to structure the breaking changes information architecture with the new doc system, let's keep it simple, as with the previous revision of this PR , with a simple one-liner pointing to the PR for each of these Breaking Changes. This is ok for a beta, and we can figure out the optimal structure and breakdown post doc feature freeze

Reply:
As per [@karenzone 's opinion](https://github.com/elastic/logstash/pull/17027#discussion_r1956373477), I have made sub-sections and tried to keep simple (single line) as much as possible. We need more @karenzone's help from now 🙏